src/screens/TopicMemberScreen/index.js

```diff
@@ -186,7 +185,7 @@ const TopicMemberScreen = () => {
         decelerationRate="fast"
         scrollEventThrottle={1}
         style={styles.fragmentContainer}
-        contentContainerStyle={[styles.fragmentContentContainer, {paddingTop: headerShowHeight}]}
+        contentContainerStyle={[styles.fragmentContentContainer]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The padding top of `fragmentContentContainer` has been removed. If this was intentional and does not affect the layout, then it's fine. However, if not, this could cause layout issues.
```diff
- contentContainerStyle={[styles.fragmentContentContainer]}
+ contentContainerStyle={[styles.fragmentContentContainer, {paddingTop: headerShowHeight}]}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->